src/keycloak/login-as-guest.js

```diff
@@ -9,8 +10,14 @@ const headers = {
 
 const loginAsGuest = async () => {
   try {
+    let clientId = apiConfig?.clientId
+    if (clientId === undefined) {
```

Comment:
We should also check for null. We could do multiple ways:
1.  if (clientId == undefined)
2. if (clientId) or if (!!clientId)